python/ray/data/_internal/logical/operators/map_operator.py

```diff
@@ -30,9 +32,11 @@ def __init__(
                 inspecting the logical plan of a Dataset.
             input_op: The operator preceding this operator in the plan DAG. The outputs
                 of `input_op` will be the inputs to this operator.
+            min_rows_per_block: The target size for blocks outputted by this operator.
```

Comment:
nit, it's not accurate to say it is the target size of outputs. Instead, it's actually the minimal num of rows for the inputs. better to name it `min_rows_per_input_bundle`.

Reply:
Yeah, I agree. Used the name for consistency with the existing parameters. Will update all of them in a follow-up PR.

IIUC, with the current implementation of `_BlockRefBundler`, `target_rows_per_input_bundle` is probably the most accurate name.

https://github.com/ray-project/ray/blob/df3dd96c1e54df5428b6a6dc3f085103421cfc89/python/ray/data/_internal/execution/operators/map_operator.py#L477-L479